src/Commands/Modules/Moderation/ClearCommand.cs

```diff
@@ -17,7 +18,17 @@ public partial class ModerationModule : BrackeysBotModule
         public async Task ClearMessagesAsync(
             [Summary("The amount of messages to clear")] int count)
         {
-            var messages = await Context.Channel.GetMessagesAsync(count + 1).FlattenAsync();
+            int maxHistory = Context.Configuration.ClearMessageMaxHistory;
+            if (count + 1 > maxHistory) 
```

Comment:
We can shorten to
```cs
count = count + 1 > maxHistory ? maxHistory : count + 1;
```

Reply:
Eeeh yea. I was not entirely happy with readability. But I still am not. Stupid newlines for braces {}

Reply:
If you don't like the ternary method you can always just leave out the braces for single-lined if-conditions, no need for newlined braces. :)

Reply:
Habit from work where we are required to do braces at all times